frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -133,14 +133,22 @@ export default ({
           this.activeComponent = link.component
         }
       }
-      this.tabNav.forEach(item => {
-        item.links.forEach(link => {
-          switchTabIfMatch(link)
-        })
+      const allTabNavLinks = this.tabNav.reduce(
+        (allLinks, item) => [...allLinks, ...item.links], []
+      )
+      const fallbackLink = allTabNavLinks.find(item => item.url === 'my-account')
```

Comment:
`fallbackLink` here is the case where the specified `tab` query doesn't match any tab items in the last. (e.g. `?modal=UserSettings&tab=asdfsadf`) we need to manually direct it to the `my-account` tab.